src/common/filesystembase.cpp

```diff
@@ -133,6 +133,11 @@ bool FileSystem::rename(const QString &originFileName,
     bool success = false;
     QString error;
 #ifdef Q_OS_WIN
+    if (FileSystem::isFileLocked(originFileName, FileSystem::LockMode::Exclusive)) {
+        *errorString = QCoreApplication::tr("FileSystem", "Can't rename %1, the file is currently in use").arg(originFileName);
```

Comment:
Attention: This is unchecked, `errorString` could be a nullptr. Other places use a dummy object in case `errorString` is nullptr. You should rather set the variable `error`